docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,228 @@
+
+```js
+const claimOwnershipPayload =
+  web3.eth.abi.encodeFunctionSignature('claimOwnership()');
+
+await newKeyManager.methods['execute(bytes)'](claimOwnershipPayload).send({
+  from: account.address,
+  gas: 1000000,
+  gasPrice: '1000000000',
+});
+```
+
+## The upgrade has been completed successfully.
+
+## Final code
+
+```javascript title="upgrade-lsp6.js"
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json' assert { type: 'json' };
+import Web3 from 'web3';
+const web3 = new Web3('https://rpc.l14.lukso.network');
+
+export const privateKey = '0x...';
+export const universalProfileAddress = '0x...';
+export const keyManagerAddress = '0x...';
+
+const upgradeLSP6 = async () => {
+  // Initialize the controller account
+  const account = web3.eth.accounts.privateKeyToAccount(privateKey);
+  web3.eth.accounts.wallet.add(account);
+
+  // Initialize your current LSP6 Key Manager
+  const oldKeyManager = new web3.eth.Contract(
+    LSP6KeyManager.abi,
+    keyManagerAddress,
+  );
+
+  // Deploy a new LSP6 Key Manager
+  const NewKeyManager = new web3.eth.Contract(LSP6KeyManager.abi);
+  const newKeyManager = await NewKeyManager.deploy({
+    data: LSP6KeyManager.bytecode,
+    arguments: [universalProfileAddress],
+  }).send({
+    from: account.address,
+    gas: 3000000,
+    gasPrice: '1000000000',
+  });
+
+  console.log(newKeyManager._address);
+
+  // Transfer the ownership of your Universal Profile from the current LSP6 Key Manager to a new LSP6 Key Manager
+  const transferOwnershipPayload =
+    web3.eth.abi.encodeFunctionSignature('transferOwnership(address)') +
+    web3.eth.abi
+      .encodeParameter('address', newKeyManager._address)
+      .substring(2);
+
+  await oldKeyManager.methods['execute(bytes)'](transferOwnershipPayload).send({
+    from: account.address,
+    gas: 1000000,
+    gasPrice: '1000000000',
+  });
+
+  // Accept the ownership of your Universal Profile from the new LSP6 Key Manager
+  const claimOwnershipPayload =
+    web3.eth.abi.encodeFunctionSignature('claimOwnership()');
+
+  await newKeyManager.methods['execute(bytes)'](claimOwnershipPayload).send({
+    from: account.address,
+    gas: 1000000,
+    gasPrice: '1000000000',
+  });
+};
+
+await upgradeLSP6();
+```
+
+## Test the new LSP6 Key Manager
+
+We can now check the owner of the Universal Profile. If everything went through, the owner should be the address of the new LSP6 Key Manager.
+Create the following file with the name `test-new-lsp6.js` and run:
+
+```shell
+node test-new-lsp6.js
+```
+
+```javascript title="test-new-lsp6.js"
+import LSP0ERC725YAccount from '@lukso/lsp-smart-contracts/artifacts/LSP0ERC725YAccount.json' assert { type: 'json' };
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json' assert { type: 'json' };
+import { keyManagerAddress, universalProfileAddress } from './constants.js';
```

Comment:
I think this constants.js file does not exist anymore

Maybe you need to put the const here too